src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
+  border: 1px solid $primary-bg-color;
+
+  transition: transform $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
```

Comment:
I see you added the zoom animation but it still doesn't work via the demo link, maybe you forgot to redeploy your site?